cds/modules/webhooks/tasks.py

```diff
@@ -62,38 +64,52 @@ def sse_publish_event(channel, type_, state, meta):
         current_sse.publish(data=data, type_=type_, channel=channel)
 
 
-def _factory_sse_task_base(type_=None):
+def _factory_avc_task_base(type_=None, undo_fun=None):
     """Build base celery task to send SSE messages upon status update.
 
     :param type_: Type of SSE message to send.
-    :return: ``SSETask`` class.
+    :return: ``AVCTask`` class.
     """
-    class SSETask(Task):
+    class AVCTask(Task):
         """Base class for tasks which might be sending SSE messages."""
 
         abstract = True
 
         def __init__(self):
-            """."""
-            super(SSETask, self).__init__()
+            """Constructor."""
+            super(AVCTask, self).__init__()
             self._base_payload = {}
 
+        def _extract_call_arguments(self, arg_list, **kwargs):
+            for name in arg_list:
+                setattr(self, name, kwargs.pop(name, None))
+            return kwargs
+
         def __call__(self, *args, **kwargs):
             """Extract SSE channel from keyword arguments.
 
             .. note ::
                 the channel is extracted from the ``sse_channel`` keyword
                 argument.
             """
-            self.sse_channel = kwargs.pop('sse_channel', None)
+            arg_list = [
```

Comment:
In general for constants we use capitals.

To make it a bit nicer, maybe we can have a list of tuples instead, `(key, default_value)`.